playbook/webpack.config.js

```diff
@@ -174,6 +174,14 @@ module.exports = {
           SASS_LOADER,
         ],
       },
+      {
+        test: /\.css$/i,
+        use: ["style-loader", "css-loader"],
+      },
+      {
+        test: /\.png$/,
+        use: 'file-loader'
+      },
```

Comment:
Do these need to be disclosed to developers? Meaning to use this kit I need to have loaders in my app correct? If so perhaps we write a quick sentence?

Reply:
@jasoncypret we tried the kit on Nitro and it was working fine. I showed this to @jasperfurniss too, and we do not think the developer would have to config any loader.